build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
To make it running on MAC, let's use below.
```bash
#!/usr/bin/env bash
```

Reply:
Hi @sunghan-chang ,

Sure, that will be fixed, thanks!